crypto/engine/eng_devcrypto.c

```diff
@@ -34,6 +35,36 @@
  * saner...  why re-open /dev/crypto for every session?
  */
 static int cfd;
+#define DEVCRYPTO_REQUIRE_ACCELERATED 0 /* require confirmation of acceleration */
+#define DEVCRYPTO_USE_SOFTWARE        1 /* allow software drivers */
+#define DEVCRYPTO_REJECT_SOFTWARE     2  /* only disallow confirmed software drivers */
+
+#define STR_(S) #S
+#define STR(S)  STR_(S)
```

Comment:
Please use `OPENSSL_MSTR` instead of your own stringification macro. (It was introduced only recently in 3a63dbef15b62b121c5df8762f8cb915fb06b27a, so you might have to rebase, I didn't check.)

https://github.com/openssl/openssl/blob/1057c2c39f5df45c36c0fc4d78dc9d3b91f78bc6/include/openssl/opensslv.h#L83-L85

@levitte is there a more suitable location for such a general purpose macro than the _version_ header file?

Reply:
(`OPENSSL_MSTR(x)` will have to be backported if @cotequeiroz is going to use it.)

Reply:
Hmm, you're right that `OPENSSL_MSTR` could have been placed in a better spot.  I think this is a discussion for a different issue or PR and shouldn't hold up this PR, so in my judgement, this nit has very low priority.

Reply:
If `opensslv.h` is visible from the module (which I guess it is) there is no need to wait for your refactoring, he could use the macro right away.

Reply:
> ... this nit has very low priority.

Agreed, if you put it onto your todo list for the refactoring to locate and replace all existing stringification macros ;-)

Reply:
BTW: the name `OPENSSL_MSTR(x)` is not very self-explaining IMHO. Couldn't you call it `OPENSSL_MAKESTRING(x)` or `OSSL_MAKESTRING(x)` (if the former is too long) instead?

Reply:
This macro is not needed in 1.1.1 code; I can use it for the master commit, and it'll be done.  If you move its definitions somewhere else, the build will just fail, which should make it pretty easy to pick up.

Reply:
You can just leave everything as it is. Richard reminded me that we can't backport the macro to 1.1.1, because it would count as a new feature. Sorry for the noise ;-)

Reply:
BTW, I picked the code as is from `crypto/s390xcap.c:20`, so if you have a todo list, you might want to add that to the list.

Reply:
> 
> 
> You can just leave everything as it is. Richard reminded me that we can't backport the macro to 1.1.1, because it would count as a new feature. Sorry for the noise ;-)

It won't be in 1.1.1; this macro is needed to assemble the cmds help, which is not going into 1.1.1.  That's why I suggested to change it.

Reply:
Do whatever suits you best.